Automattic/vip-go-mu-plugins
# Featured Plugins Fix: Wait for images to load before calculating height

Currently if the JS loads prior to the featured plugin images, the box height will not be calculated properly.  This sets the initial calculation to happen after the images have loaded.

Edit:  I'm now trying an approach using CSS Grid so we don't need to calculate anything.

## Description

Currently if the images load after the script calculation happens:
<img width="1145" alt="Screen Shot 2021-05-28 at 8 53 57 AM" src="https://user-images.githubusercontent.com/3220162/120021017-5e6f7480-bf9f-11eb-820e-d3204fbf02fd.png">

This PR just moves the initial calculation to be after the window has loaded so that the calculation happens after images are loaded.

## Changelog Description
### Fix featured partner plugin height calculation 

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test
Example:

1. Check out the PR
2. Make sure to change the version number in the `vip-plugins.php` file to invalidate the cache
3. Throttle your browser to something like "Slow 3G" to get consistent results
4. See if the calculation is accurate
